camille/inputs/bazefetcher.py

```diff
@@ -0,0 +1,41 @@
+from pandas import DataFrame, Timestamp, read_json
+from os import path, walk, listdir
+import dateutil
+
+def _bazefetcher_time_filter(config, filename):
+    filename = path.split(filename)[-1]
+    filename = path.splitext(filename)[0]
+
+    try:
+        s, e = filename.split('_')[-2:]
+    except ValueError:
+        return False
+
+    start = Timestamp(dateutil.parser.parse(s.replace('.',':')))
+    end = Timestamp(dateutil.parser.parse(e.replace('.',':')))
+
+    return config['start'] <= start and end < config['end']
+
+def load(config, base_folder='./'):
+    df = DataFrame()
+    for i, t in enumerate(config['tag']):
+        dirs = [ path.join(base_folder, d)
+                 for d in listdir(base_folder) if  t.match(d) ]
+        files = [ list(map( lambda x: path.join(d, x), listdir(d) ))
+                  for d in dirs ]
+        files = sum( files, [] )
+        files = filter( lambda x: _bazefetcher_time_filter(config, x), files )
```

Comment:
I like filter+lambda, but I have been told that it is non-pythonic. Supposedly the better alternative is a list-comprehension with a `if` inside: 

```python
[x for x in files if _bazefetcher_time_filter(config, x)]